docs/topics/crud-mission-integration-testing.adoc

```diff
@@ -2,7 +2,7 @@
 = Running Integration Tests
 
 This booster contains a set of integration tests.
-To run them, you must be connected to an OpenShift instance and select the project that contains your CRUD application and database.
+To run them, you must be connected to an OpenShift instance and select the project that contains your Relational Database Backend application and database.
```

Comment:
This looks weird. The application is backed by a relational database, sure, but it's not a "relational database backend" for the frontend or anything. Maybe just "select the project that contains your application and database"?